### PR TITLE
Enhance example 55

### PR DESCRIPTION
**Pull request by t-bltg**
pr1_ex

@BeastyBlacksmith, after polishing does it seem ok to you ?

### CODE REVIEW

File: src/examples.jl [hunk 1]
```diff
@@ -1165,24 +1165,19 @@ const _examples = PlotExample[
...
-		  sinc(r / pi)
-		end
+                x, y = collect(-6:0.5:10), collect(-8:0.5:8)
 
-                args = (x, y, f)
+                args = x, y, (x, y) -> sinc(norm([x, y]) / π)
                 kwargs = Dict(
                     :xlabel => "x", :ylabel => "y", :zlabel => "z",
-                    :grid => true, :minorgrid => true, :dpi => 200
+                    :grid => true, :minorgrid => true, :dpi => 300
```
Comment (t-bltg) [inline src/examples.jl:1180]: Not sure about the dpi in the docs, we'll see once it's fully rendered by `Documenter.jl`...

File: src/examples.jl [hunk 2]
```diff
@@ -1209,7 +1204,10 @@ const _examples = PlotExample[
                     ))
                 end
 
-                plot(plots..., layout=(@layout [_ ° _; ° ° °; ° ° °]), margin=2Plots.mm)
+                plt = plot(plots..., layout=(@layout [_ ° _; ° ° °; ° ° °]), margin=-1Plots.mm)
```
Comment (BeastyBlacksmith) [inline src/examples.jl:1207]: why do you have a negative margin here?

Comment (t-bltg) [reply src/examples.jl:1207]: To make the final plot more compact.

File: src/examples.jl [hunk 3]
```diff
@@ -1174,7 +1174,7 @@ const _examples = PlotExample[
 
                 x, y = collect(-6:0.5:10), collect(-8:0.5:8)
 
-                args = x, y, (x, y) -> sinc(norm([x, y]) / π)
+                args = x, y, (x, y) -> (r = norm([x, y]) + eps(); sin(r) / r)
```
Comment (BeastyBlacksmith) [inline src/examples.jl:1177]: what is the issue with `sinc`?

**Comment by t-bltg** [comment]
@BeastyBlacksmith, finally managed to generate a new img with `sinc` :exploding_head:, CI passes.

I everything ok now ?

**Comment by BeastyBlacksmith** [comment]
Great! Thank you.

**Pull request merged**